### PR TITLE
changes to work with clisops split method and filenamer

### PR DESCRIPTION
**Pull request by ellesmith88**

### CODE REVIEW

**Comment by cehbrecht** [comment]
@ellesmith88 I think daops is not using the clisops master. Should I update?

**Comment by ellesmith88** [comment]
@cehbrecht Oh yes, you're right. Yes please :)

**Comment by cehbrecht** [comment]
@ellesmith88 there are some more "prints" in daops:
https://github.com/roocs/daops/blob/89f3b8cffb0e84020e56c6af9450601300aae8c1/daops/utils/normalise.py#L7

Should they be converted to use logging?

**Review by cehbrecht — APPROVED** [review]

File: daops/utils/normalise.py [hunk 1]
```diff
@@ -22,5 +23,10 @@ def __init__(self, inputs=None):
         self.file_paths = []
 
     def add(self, dset, result):
+        print(result)
```
[the remark this answers is not in the document]
Comment (cehbrecht) [reply daops/utils/normalise.py]: done.

**Comment by ellesmith88** [comment]
@cehbrecht Yes I suppose they should be

**Comment by cehbrecht** [comment]
> @cehbrecht Yes I suppose they should be
Ok. I make an update and merge then ...

**Pull request merged**